### ISSUE TITLE
`no-bare-strings` doesn't seem to lint handlebars correctly

### ISSUE THREAD

**Issue by billybonks** [ember-template-lint/ember-template-lint#99]
given the following rule

``` javascript
    'bare-strings':{
      whitelist: ['(', ')', ',', '.', '&', '+', '-', '=', '*', '/', '#', '%', '!', '?', ':', '[', ']', '{', '}'],
      globalAttributes: ['label', 'title'],
      elementAttributes: {
        input: [ 'placeholder' ],
        img: [ 'alt' ]
      }
    }
```

`<div title="something">`
`<div label="something>`

will throw an error similar to this 

```
bare-strings: Non-translated string used in `title` attribute (hospitalrun/appointments/search/template @ L2:C32):`something`

```

I would like these rules to apply to handlebars as well

 `{{em-select class="col-sm-3" label="I AM SUCH A BARE STRING OMG" property="selectedAppointmentType" content=visitTypesWithEmpty}}` 

this should throw the same error, but it doesn't at the moment

**Comment by rwjblue** [comment]
The issue here, is that the linter cannot tell what you are doing with that value passed into the component.  For all I know, you are passing that string as a key to a translation tool in the `em-select` component.

**Comment by billybonks** [comment]
Maybe we can make it just enforce Sunday expressions.

**Comment by billybonks** [comment]
Sub*

**Comment by bertdeblock** [comment]
Is this issue still relevant? `bare-strings` was also renamed to `no-bare-strings` in `v0.8.16`.

**Comment by locks** [comment]
Closing this issue as no longer relevant!
